tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/BRunUtil.java

```diff
@@ -1357,13 +1360,16 @@ private static String getConsoleOutput(InputStream inputStream) {
     public static void runInit(CompileResult compileResult)
             throws ClassNotFoundException {
         PackageManifest packageManifest = compileResult.packageManifest();
-        String initClassName = JarResolver.getQualifiedClassName(packageManifest.org().toString(),
-                packageManifest.name().toString(),
-                packageManifest.version().toString(),
-                MODULE_INIT_CLASS_NAME);
+        String org = packageManifest.org().toString();
+        String module = packageManifest.name().toString();
+        String version = packageManifest.version().toString();
+        String initClassName = JarResolver.getQualifiedClassName(org, module, version, MODULE_INIT_CLASS_NAME);
+        String configClassName = JarResolver.getQualifiedClassName(org, module, version, CONFIGURATION_CLASS_NAME);
 
         Class<?> initClazz = compileResult.getClassLoader().loadClass(initClassName);
         final Scheduler scheduler = new Scheduler(false);
+        directRun(compileResult.getClassLoader().loadClass(configClassName), "$configureInit", new Class[]{Path.class},
```

Comment:
AFAIU Since we move the current module init into config init this change was required. @manuranga This should be ok I think.

Reply:
Ya but that's not the only reason, we'll need this anyway to support config toml in unit tests.
@HindujaB with this let's try to have unit tests for our next task instead integration.